grpc/src/test/java/com/linecorp/armeria/common/grpc/GrpcExceptionHandlerFunctionBuilderTest.java

```diff
@@ -140,17 +144,19 @@ void mapStatusAndMetadata() {
                         })
                         .build();
 
-        final Status status = Status.UNKNOWN.withCause(new B1Exception());
-
+        final B1Exception cause = new B1Exception();
         final Metadata metadata1 = new Metadata();
-        final Status newStatus1 = GrpcStatus.fromExceptionHandler(exceptionHandler, ctx, status, metadata1);
+        final Status newStatus1 = exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault())
+                                                  .apply(ctx, cause, metadata1);
         assertThat(newStatus1.getCode()).isEqualTo(Code.ABORTED);
         assertThat(metadata1.get(TEST_KEY)).isEqualTo("B1Exception");
         assertThat(metadata1.keys()).containsOnly(TEST_KEY.name());
 
         final Metadata metadata2 = new Metadata();
         metadata2.put(TEST_KEY2, "test");
-        final Status newStatus2 = GrpcStatus.fromExceptionHandler(exceptionHandler, ctx, status, metadata2);
+        final Status newStatus2 = exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault())
+                                                  .apply(ctx, cause, metadata2);
```

Comment:
```suggestion
        final Status newStatus2 = exceptionHandler.apply(ctx, cause, metadata2);
```

Reply:
Applied feedback, Thanks.
I've checked that test is working fine after change.